config/schedule.rb

```diff
@@ -1,3 +1,6 @@
+rails = File.join Dir.getwd, "config", "environment.rb"
+require rails
```

Comment:
Is there a way we could point directly to the path? Then we don't need the rails helper. It should all be fine on `localhost`, the only issue I could see is the port number.